### ISSUE TITLE
balloon support for KVM (PATCH)

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#229]
Originally reported of Google Code with ID 173.

```
 A patch against the devel-4.2                                                                                                                                                                  
<http://code.google.com/p/ganeti/wiki/SourceCode> branch of ganeti was                                                                                                                                      
implemented to support the balloon                                                                                                                                                                          
<http://www.linux-kvm.org/page/KVM_Features> memory driver KVM                                                                                                                                              
<http://www.linux-kvm.org/>. In order to preserve backward                                                                                                                                                  
compatibility, the *-balloon* option is not added unless the *balloon*                                                                                                                                      
hypervisor parameter is explicitly specified. The KVM may be compiled to                                                                                                                                    
activate ballooning by default or not and forcing the option would                                                                                                                                          
change this behavior and produce unexpected results.   


```
Originally added on 2011-07-05 06:58:21 +0000 UTC.

Attachments: 
[balloon.patch](https://storage.googleapis.com/google-code-attachments/ganeti/issue-173/comment-0/balloon.patch)

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in 2.6, with max-min mem support in Ganeti.
Thanks a lot,

Guido
```
Originally added on 2012-02-11 18:23:06 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Migrate old issues from 'fixed' to 'released'.
```
Originally added on 2012-10-22 09:16:17 +0000 UTC.

Changed State: Released
